### DISCUSSION TITLE
Any restrictions with Bankamp Lights?

### DISCUSSION THREAD

**Discussion by ahemwe** [Koenkk/zigbee2mqtt#14177] [Devices]
I consider buying a Bankamp light which has a poorly documented zigbee feature.

German documentation, which mentions that the lights are hue compatible:
- https://www.bankamp.de/technologie/zigbee/
- https://www.bankamp.de/media/d0/4a/d5/1619432862/ZigBee%20Anleitung%20L6090_1_BOOK.pdf

I did not find any Bankamp devices listed for zigbee2mqtt and also no issue in the project which refers to this vendor.
After looking in the documentation, I'd try to follow the how to in order to get these lamps supported. 

Since lights of this vendor are quite expensive and having zigbee is an important feature, I just want to ask if there are any problems to be expected. It would be very bad if light of this vendor are somehow incompatible with zigbee2mqtt.

Any experience, concerns or information on that topic?

**Comment by Koenkk** [comment]
No guarantees but I don't for see any problems. Lights are often very easy to add, just fill in https://github.com/Koenkk/zigbee2mqtt/issues/new?assignees=&labels=new+device+support&template=new_device_support.yaml&title=%5BNew+device+support%5D%3A+ and I can add it.

**Reply by ahemwe** [reply]
Thanks. I've ordered it and will follow your guide to get it running with zigbee2mqtt.

**Comment by Brutuss** [comment]
I also have a Bankamp Light 2214/2-36 its being recognized as a Trust ZLED-2709.

On and off function works great but the dimming not so good. Example:

Slider to 10% (Lamp not changing)
Slider to 20% (Lamp to 10%)
Slider to 50% (Lamp to 20%) 
Lamp Off (brightness changes to 50% and then off).

What can I do to fix this?

**Reply by ahemwe** [reply]
I've added a Bankamp configuration in https://github.com/Koenkk/zigbee-herdsman-converters/pull/5369
The two lights I own are working with this configuration and quite good.

I plan to update the description in a new PR which mentions two things:
- how to reset the lights by means of a wall switch
- that binding with tradfri and hue remotes works on my end

**Reply by Brutuss** [reply]
Is yours recognized as a Bankamp light? 

Added the bankamp.js as an external converter file from the zigbee-herdsman-converters 
Reset the lamp still it shows as a Trust lamp and reacts as mentioned before. 

I think because its not recognized as an bankamp light its not working for me. 
How to fix this?

**Reply by wschomburg** [reply]
Hi @Brutuss, 
Have you solved the wrong identification? I've a Bankamp 2157/155-33 STRADA and it's also identified as Trust ZLED-2709. The dimming doesn't work well with it as well. I already tried to create an external converter for it with luck.

**Reply by Brutuss** [reply]
Hi @wschomburg, the lamps weren't reliable when controlling them with zwave. I send them back to the shop and got my money back. To bad because they were nice lamps.
